src/blueprints/tv2/tv2-blueprint-configuration-validator.ts

```diff
@@ -39,4 +42,17 @@ export class Tv2BlueprintConfigurationValidator implements BlueprintValidateConf
         })
     })
   }
+
+  private validateShowStyleVariants(showStyleVariants: ShowStyleVariant[]): StatusMessage[] {
+    return showStyleVariants
+      .filter(variant => !(variant.blueprintConfiguration as CoreShowStyleVariantBlueprintConfiguration).GfxDefaults)
```

Comment:
In addition to `GfxDefaults` not being a defined attribute, shouldn't blueprints generate a status message if the `GfxDefaults` is an empty array?

Reply:
We could also take it all the way to validating that you have picked at valid combination of Schema/design depending on the Graphic setup. But I would prefer to do the complex validation for once the settings are defined by SofieWebClient/SofieServer.

I can add the extra check for empty array now if you want.

Reply:
As long as the checks we make ensures that the value conforms to the type, then I am satisfied.
I can see that the length is checked where it is used, so I'll leave it up to you to add it or not. A minor thing is that we only check `GfxDefaults` to be a truthy value, but not that it is an array. A suggestion here could be to check if it is an Array, e.g.
```typescript
variant => !Array.isArray((variant.blueprintConfiguration as CoreShowStyleVariantBlueprintConfiguration).GfxDefaults)
```

Reply:
Updated to check that it is an array.